CMakeLists.txt

```diff
@@ -1,30 +1,30 @@
-cmake_minimum_required(VERSION 3.13)
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
 
-project(NovelRT VERSION 0.0.1)
+project(NovelRT
+  VERSION 0.1.0
+  DESCRIPTION "NovelRT game engine"
+  HOMEPAGE_URL "https://github.com/novelrt/NovelRT"
+  LANGUAGES C CXX
+)
 
-set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
-set(CMAKE_CXX_STANDARD 17)
+list(APPEND CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake/modules")
 
-if (MSVC)
```

Comment:
nit: extraneous space

(this is a reminder to remind me to fix it later)

Reply:
wait i'm stupid this is a removed line


this is what i get for commenting before bed